client/core/core.go

```diff
@@ -1207,6 +1210,42 @@ func (c *Core) handleUnbookOrderMsg(dc *dexConnection, msg *msgjson.Message) err
 	return ob.Unbook(&note)
 }
 
+// handleEpochOrderMsg is called when an epoch_order notification is
+// received.
+func (c *Core) handleEpochOrderMsg(dc *dexConnection, msg *msgjson.Message) error {
+	var note msgjson.EpochOrderNote
+	err := json.Unmarshal(msg.Payload, &note)
+	if err != nil {
+		return fmt.Errorf("epoch order note unmarshal error: %v", err)
+	}
```

Comment:
This is fine like you have it, but I wanted to make sure you knew about the new `(*Message).Unmarshal`. 

https://github.com/dnldd/dcrdex/blob/f8579cdc33690386df0917d152b716f641418201/dex/msgjson/types.go#L299-L302